framework/src/org/checkerframework/common/value/ValueChecker.java

```diff
@@ -10,8 +10,8 @@
  * The Constant Value Checker is a constant propagation analysis: for each variable, it determines
  * whether that variable's value can be known at compile time.
  *
- * <p>The Constant Value Checker has no dependencies, but it does trust {@link
- * org.checkerframework.checker.index.qual.Positive} annotations from the {@link
+ * <p>The Constant Value Checker has no dependencies, but it does trust {@code
```

Comment:
You should be able to link to things that are on the classpath.  (So long as it is fully qualified.)